CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/1.1.0...HEAD)
+### Feature
+* **Spark: Publish the `ProcessingEngineRunFacet` as part of the normal operation of the `OpenLineageSparkEventListener`.** [`#2089`](https://github.com/OpenLineage/OpenLineage/pull/2089) [@d-m-h](https://github.com/d-m-h)
+  *Publishes the spec-defined `ProcessEngineRunFacet` alongside the custom `SparkVersionFacet`.* 
```

Comment:
I think we should mark `SparkVersionFacet` deprecated and mention OL version where we will get rid of it (like two versions ahead).

Reply:
Done. Updated the change long as well.